### PR TITLE
issue #147: falco, falcosidekick and falco-exporter configuration

### LINKED ISSUE #147: Deploy and configure Falco
## Why

Falco is a graduated project from the CNCF and the only project to actively focuses on surveillance and alerting at the kernel level using eBPF. Specifically, Falco focuses on Runtime Security; it is designed to detect anomalous activity and potentially malicious behavior at the system level in real-time.

What is KubeArmor? KubeArmor steps in to actively block and mitigate threats. 

In our case, maybe we can select only one of the two solution. 

## TODO:

- [ ] Deploy Falco
- [ ] Configure Falco
- [ ] Configure Falco dashboards with Grafana

### PR DESCRIPTION
**Pull request by ThomasCardin**
- Closes #147

### CODE REVIEW

**Comment by rngadam** [comment]
the issue does not really document why Falco vs other solutions and how it is part of our wider strategy to secure the infrastructure.

**Comment by ThomasCardin** [comment]
@rngadam Sorry, just updated the description of the Falco issue with more informations

File: kubernetes/aks/system/falco/argo-app.yaml [hunk 1]
```diff
@@ -0,0 +1,89 @@
+---
```
Comment (SonOfLope) [inline kubernetes/aks/system/falco/argo-app.yaml:1]: missing reference in apps/argo-apps/kustomization.yaml

**Review by SonOfLope — CHANGES_REQUESTED** [review]

File: kubernetes/aks/system/falco/argo-app.yaml (hunk 1 above)
Comment (ThomasCardin) [reply kubernetes/aks/system/falco/argo-app.yaml:1]: The pull request is in draft.

**Pull request merged**